docs/meta-box.md

```diff
@@ -52,33 +52,32 @@ Ideally, this could be done at instantiation of the editor and help simplify thi
 
 When rendering the Gutenberg Page, the metaboxes are rendered to a hidden div `#metaboxes`.
 
-*The Redux store by default will hold all meta boxes as inactive*. When
+*The Redux store will hold all meta boxes as inactive by default*. When
 `INITIALIZE_META_BOX_STATE` comes in, the store will update any active meta box areas by setting the `isActive` flag to `true`. Once this happens React will check for the new props sent in by Redux on the `MetaBox` component. If that `MetaBox` is now active, instead of rendering null, a `MetaBoxArea` component will be rendered. The `MetaBox` component is the container component that mediates between the `MetaBoxArea` and the Redux Store. *If no meta boxes are active, nothing happens. This will be the default behavior, as all core meta boxes have been stripped.*
 
 #### MetaBoxArea Component
 
-When the component renders it will store a ref to the metaboxes container, retrieve the metaboxes HTML from the prefetch location.
+When the component renders it will store a reference to the metaboxes container and retrieve the metaboxes HTML from the prefetch location.
 
-When the post is updated, only meta boxes areas that are active will be submitted. This removes any unnecessary requests being made. No extra revisions, are created either by the meta box submissions. A Redux action will trigger on `REQUEST_POST_UPDATE` for any active meta box. See `editor/effects.js`. The `REQUEST_META_BOX_UPDATES` action will set that meta boxes' state to `isUpdating`, the `isUpdating` prop will be sent into the `MetaBoxArea` and cause a form submission.
+When the post is updated, only metabox areas that are active will be submitted. This prevents unnecessary requests. No extra revisions are created by the meta box submissions. A Redux action will trigger on `REQUEST_POST_UPDATE` for any active meta box. See `editor/effects.js`. The `REQUEST_META_BOX_UPDATES` action will set that meta box's state to `isUpdating`. The `isUpdating` prop will be sent into the `MetaBoxArea` and cause a form submission.
```

Comment:
In all other cases, I see "Meta Box" referred to as two separate words, e.g.

https://developer.wordpress.org/plugins/metadata/custom-meta-boxes/#what-is-a-meta-box

Should this change to "metabox" not be made?